apps/meteor/tests/end-to-end/api/livechat/00-rooms.ts

```diff
@@ -335,6 +335,29 @@ describe('LIVECHAT - rooms', () => {
 			expect(body.rooms.every((room: IOmnichannelRoom) => room.open)).to.be.true;
 			expect(body.rooms.find((froom: IOmnichannelRoom) => froom._id === room._id)).to.be.undefined;
 		});
+
+		describe('roomName filter', () => {
+			let room1: IOmnichannelRoom;
+			let room2: IOmnichannelRoom;
+			before(async () => {
+				const visitor = await createVisitor(undefined, 'TEST_1');
+				const visitor2 = await createVisitor(undefined, 'TEST_2');
+				room1 = await createLivechatRoom(visitor.token);
+				room2 = await createLivechatRoom(visitor2.token);
+			});
+
+			it('should return only rooms matching exact term when roomName is between quotes', async () => {
```

Comment:
Add a case that returns empty for `"TEST_"` when `TEST_1` is present because of the exact search.